workflows/bacterial_base.nf

```diff
@@ -46,15 +46,7 @@ workflow CALL_BACTERIAL_BASE {
         bwa_mem_ref(ch_reads, genomeReferenceDir)
         samtools_index_ref(bwa_mem_ref.out.bam)
 
-        bwa_mem_ref.out.bam
-            .join(samtools_index_ref.out.bai)
-            .multiMap { id, bam, bai -> 
-                bam: tuple(id, bam)
-                bai: bai
-            }
-            .set{ post_align_qc_ch }
-
-        post_align_qc(post_align_qc_ch.bam, post_align_qc_ch.bai, coreLociBed)
```

Comment:
I'm not following what is happening here (the removal of line 49-56), but assuming it is as expected ...

Reply:
I was converting bwa_mem_ref.out.bam into a channel containing (id, bam, bai). But the post_align_qc module only needs the bam (which was already its own channel in the first place (bwa_mem_ref.out.bam). So I deleted the unnecessary fluff.